components/automate-cli/cmd/chef-automate/iam.go

```diff
@@ -165,8 +165,6 @@ func display(v *iam_common.Version) string {
 	switch x {
 	case vsn{Major: iam_common.Version_V2, Minor: iam_common.Version_V1}:
 		return "v2.1"
-	case vsn{Major: iam_common.Version_V2, Minor: iam_common.Version_V0}:
-		return "v2.0"
```

Comment:
Let's not remove this for now. Worst case if 2.0 gets passed to this by mistake it will report 1.0 which will be super hard to debug. Until we completely kill the flags `Major: iam_common.Version_V2, Minor: iam_common.Version_V0` from the system let's leave this as it was imo.